lib/jwt_sessions.rb

```diff
@@ -89,8 +87,34 @@ def algorithm
     @algorithm ||= DEFAULT_ALGORITHM
   end
 
+  def token_store=(args)
+    @token_store =
+      if args.is_a?(Array)
+        StoreAdapters.build_by_name(args.first, args.last)
+      elsif args
+        StoreAdapters.build_by_name(args)
+      end
+  rescue NameError => e
+    raise e.class, "Token store adapter for :#{Array(args).first} was not found", e.backtrace
+  end
+
   def token_store
-    RedisTokenStore.instance(redis_url, token_prefix)
+    unless instance_variable_defined?(:@token_store)
+      begin
+        self.token_store = :redis
+      rescue LoadError
+        warn <<~MSG
+          Warning! JWTSessions uses in-memory token store.
+          Since you haven't configured JWTSessions.token_store, we tried to use Redis but couldn't load the 'redis' gem.
+
+          Make sure that 'redis' gem is present in your Gemfile (or configure a different store).
+        MSG
```

Comment:
It looks almost like we're blaming users for not having redis installed :D
Let's add a more polite message, something like
```
Warning! JWTSessions uses in-memory token store.
Unless token store is not specified explicitly, JWTSessions uses Redis by default and fallbacks to in-memory token store.

Make sure that 'redis' gem is present in your Gemfile (or configure a different store).
```

Reply:
yep, changed the wording, thanks :)